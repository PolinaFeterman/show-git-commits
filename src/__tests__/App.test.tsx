import React from "react";
import { render, fireEvent } from "@testing-library/react"; //waitForElement

//import App from "../App";

describe("<App />", () => {
  test("should display a blank login form, with remember me checked by default", async () => {
    // ???
  });
});