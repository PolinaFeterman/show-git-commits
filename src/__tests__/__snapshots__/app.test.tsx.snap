// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="app"
>
  <div
    className="header"
  >
    <h3>
       Commit Feed 
    </h3>
  </div>
  <form
    onSubmit={[Function]}
  >
    <div
      className="input"
    >
      <label
        className="input label"
      >
        Organization:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Repo: 
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Commits per page:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="number"
        value="10"
      />
    </div>
    <div
      className="button"
    >
      <button
        className="rounded"
        disabled={true}
        type="submit"
      >
        Submit
      </button>
    </div>
  </form>
</div>
`;

exports[`should render correctly 1`] = `
<div
  className="app"
>
  <div
    className="header"
  >
    <h3>
       Commit Feed 
    </h3>
  </div>
  <form
    onSubmit={[Function]}
  >
    <div
      className="input"
    >
      <label
        className="input label"
      >
        Organization:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Repo: 
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Commits per page:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="number"
        value="10"
      />
    </div>
    <div
      className="button"
    >
      <button
        className="rounded"
        disabled={true}
        type="submit"
      >
        Submit
      </button>
    </div>
  </form>
</div>
`;

exports[`should reneder correctly 1`] = `
<div
  className="app"
>
  <div
    className="header"
  >
    <h3>
       Commit Feed 
    </h3>
  </div>
  <form
    onSubmit={[Function]}
  >
    <div
      className="input"
    >
      <label
        className="input label"
      >
        Organization:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Repo: 
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="input"
    >
      <label
        className="label"
      >
        Commits per page:
      </label>
      <input
        className="rounded"
        onChange={[Function]}
        type="number"
        value="10"
      />
    </div>
    <div
      className="button"
    >
      <button
        className="rounded"
        disabled={true}
        type="submit"
      >
        Submit
      </button>
    </div>
  </form>
</div>
`;
